mithril-stm/benches/stm.rs

```diff
@@ -70,15 +67,97 @@ where
         .collect::<Vec<_>>();
 
     let clerk = StmClerk::from_signer(&signers[0]);
-    let msig = clerk.aggregate(&sigs, &msg).unwrap();
 
     group.bench_function(BenchmarkId::new("Aggregation", &param_string), |b| {
         b.iter(|| clerk.aggregate(&sigs, &msg))
     });
+}
 
-    group.bench_function(BenchmarkId::new("Verification", &param_string), |b| {
-        b.iter(|| msig.verify(&msg, &clerk.compute_avk(), &params).is_ok())
-    });
+fn batch_benches<H>(
```

Comment:
How do we get the size of benches for the batch version? I couldn't see how we get the output given in the PR description.

Reply:
What do you mean the size of benches?

Reply:
If you run `cargo bench` you get the output presented in the PR description for STM signatures. If you also want to run the benches for the multi-sig (a module which is not exposed) you need to use the feature benchmark-internals, `cargo bench --features benchmark-internals`. Does this solve the concern?